keyboards/sergiopoverony/creator_pro/keymaps/via/creator_pro.json

```diff
@@ -0,0 +1,34 @@
+{

```

Comment:
This file doesn't belong in this repo.  It needs to be removed and PR'ed to the `the-via/keyboards` repo, instead.

Reply:
Yes. i remove this.